### ISSUE TITLE
Show worker name on build page

### ISSUE THREAD

**Issue by evanchaoli** [concourse/concourse#5795]
## Summary

@vito As we discussed in the meeting, I was going to reopen #4519 to track the story, but the PR had been deleted, no way to reopen, thus I create a new issue. Please advice how to proceed.

## Context

Our worker VMs are distributed to multiple vSphere clusters, to debug some intermittent failures, e.g. transient network breakages, we have to know on which worker a step ran on, so that we can further identify on which vSphere cluster the failure occurred.

But now it's very hard to find out worker name, thus we want to request a feature to show worker name on build page.

* RFC: concourse/rfcs#
* Prior discussion: #4519
* Depends on #
* Part of #
